controller.go

```diff
@@ -794,21 +795,13 @@ func (c *controller) NewNetwork(networkType, name string, id string, options ...
 	// From this point on, we need the network specific configuration,
 	// which may come from a configuration-only network
 	if network.configFrom != "" {
-		t, err := c.getConfigNetwork(network.configFrom)
+		t, err = c.getConfigNetwork(network.configFrom)
```

Comment:
why do we need this change ? 
old cold also does the same without declaring variable at the beginning, right ?